### ISSUE TITLE
[Bug]: SpikeGLXLFPInterface saves data with wrong name

### ISSUE THREAD

**Issue by bendichter** [catalystneuro/neuroconv#287]
### What happened?

SpikeGLXLFPInterface saves a timeseries with the name "ElectricalSeriesRaw" despite metadata suggesting it would save as "ElectricalSeriesLFP".

```python
import os
from neuroconv.datainterfaces import SpikeGLXLFPInterface

ECEPHY_DATA_PATH = "/Users/bendichter/data/neuroconv_testing_data/ephy_testing_data"

folder_path = f"{ECEPHY_DATA_PATH}/spikeglx/Noise4Sam_g0"
dir_name = os.path.split(folder_path)[1]
lf_path = os.path.join(folder_path, f"{dir_name}_imec0", f"{dir_name}_t0.imec0.lf.bin")

lf_inter = SpikeGLXLFPInterface(lf_path)
metadata = lf_inter.get_metadata()
print(metadata)
lf_inter.run_conversion("lf_out5.nwb", metadata=metadata)
```

metadata:
```
{'NWBFile': {'session_description': 'Auto-generated by neuroconv',
  'identifier': '4b617739-050b-4c6b-ab51-093a3ffa64a8',
  'session_start_time': datetime.datetime(2020, 11, 3, 10, 35, 10)},
 'Ecephys': {'Device': [{'name': 'Neuropixel-Imec',
    'description': '{"probe_type": "0", "probe_type_description": "NP1.0", "flex_part_number": "NP2_FLEX_0", "connected_base_station_part_number": "NP2_QBSC_00"}',
    'manufacturer': 'Imec'}],
  'ElectrodeGroup': [{'name': 's0',
    'description': 'a group representing shank s0',
    'location': 'unknown',
    'device': 'Neuropixel-Imec'}],
  'Electrodes': [{'name': 'shank_electrode_number',
    'description': '0-indexed channel within a shank.'},
   {'name': 'group_name',
    'description': 'Name of the ElectrodeGroup this electrode is a part of.'},
   {'name': 'contact_shapes', 'description': 'The shape of the electrode'}],
  'ElectricalSeriesLFP': {'name': 'ElectricalSeriesLFP',
   'description': 'LFP traces for the processed (lf) SpikeGLX data.'}}}
```

This metadata would suggest that the name of the ElectricalSeries will be "ElectricalSeriesLFP" but it is not. It is "ElectricalSeriewsRaw".

<img width="210" alt="image" src="https://user-images.githubusercontent.com/844306/215285812-5d42c9e4-011b-4d55-a072-6aa18de5dd10.png">

### Steps to Reproduce

```python
see above
```

### Operating System

macOS

### Python Executable

Conda

### Python Version

3.7

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

**Comment by bendichter** [comment]
Now I'm getting this error and I'm not sure why the constructor is not working anymore:

```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In [1], line 10
      7 dir_name = os.path.split(folder_path)[1]
      8 lf_path = os.path.join(folder_path, f"{dir_name}_imec0", f"{dir_name}_t0.imec0.lf.bin")
---> 10 lf_inter = SpikeGLXLFPInterface(lf_path)
     11 metadata = lf_inter.get_metadata()
     12 print(metadata)

File ~/dev/neuroconv/src/neuroconv/baseextractorinterface.py:40, in BaseExtractorInterface.__new__(cls, *args, **kwargs)
     39 def __new__(cls, *args, **kwargs):
---> 40     cls.Extractor = getattr(cls, "Extractor")
     41     return object.__new__(cls)

File ~/dev/neuroconv/src/neuroconv/baseextractorinterface.py:12, in _LazyExtractorImport.__getattribute__(self, name)
     10 if name == "Extractor" and super().__getattribute__("Extractor") is None:
     11     extractor_module = get_package(package_name=super().__getattribute__("ExtractorModuleName"))
---> 12     extractor = getattr(
     13         extractor_module,
     14         super().__getattribute__("ExtractorName") or self.__name__.replace("Interface", "Extractor"),
     15     )
     16     return extractor
     17 return super().__getattribute__(name)

AttributeError: module 'spikeinterface.extractors' has no attribute 'SpikeGLXLFPExtractor'
```

**Comment by CodyCBakerPhD** [comment]
First problem related to metadata is beacuse it's not harnessing the correct `es_key` in the metadata. If you use `run_conversion(..., es_key="ElectricalSeriesLFP")` it would solve the issue, but it should know to do that by default anyway so I went and fixed that.

Second problem related to construction - did you happen to initialize a SpikeGLXRecordingInterface in the same environment prior to re-running it on LFP only? Either way I'm pretty sure the 'correct' fix in #288 will resolve it, just curious as to how it was detected.
